RevenueCat/Plugins/Editor/RevenueCatDependencies.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <dependencies>
-    <androidPackages>
+    <!-- <androidPackages>
```

Comment:
To avoid having to do an Android release, and since the developers are not using EDM4U in their Android builds, I decided to comment this out and to include modified `.aar` files of purchases-android built from this branch https://github.com/RevenueCat/purchases-android/pull/461